packages/compiler-cli/src/ngtsc/annotations/src/component.ts

```diff
@@ -407,8 +423,12 @@ export class ComponentDecoratorHandler implements
       }
     }
 
-    let inlineStyles: string[]|null = null;
-    if (component.has('styles')) {
+    // If inline styles were preprocessed use those
+    let inlineStyles = this.preanalyzeStylesCache.get(node) || null;
+    if (inlineStyles !== null) {
```

Comment:
Oh but actually, the compiler is in the habit of an explicit `has` call before calling `get`:

```suggestion
    let inlineStyles: string|null = null;
    // If inline styles were preprocessed use those
    if (this.preanalyzeStylesCache.has(node)) {
      inlineStyles = this.preanalyzeStylesCache.get(node)!;
```

Reply:
That requires a double lookup and a non-null assertion though.

Reply:
Yes; we've found that to not add any noticeable overhead though.